invenio_rdm_records/records/api.py

```diff
@@ -80,6 +83,10 @@ class RDMRecord(CommonFieldsMixin, Record):
         delete=False,
     )
 
+    has_draft = HasDraftCheckField(LocalProxy(lambda: RDMDraft))
+
+    is_record = True
```

Comment:
I understand we need to know if this is a draft or record, but is it not implicit via the class, index or other information we already have available?

Reply:
Or, e.g. Draft records doesn't have a ``has_draft`` attribute.